### PR TITLE
lang/erlang-runtime25: update to version 25.2.1

### PR DESCRIPTION
**Pull request by sax**

### CODE REVIEW

**Comment by sax** [comment]
This successfully builds, and I've successfully run elixir 1.14.3 using the resulting package. I still need to build all the packages that depend on lang/erlang to verify that they work against these changes.

**Comment by sax** [comment]
I see the following ports effecting by these changes:

```
$ pkg rquery %ro lang/erlang-runtime25
databases/couchdb3
```

```
$ pkg rquery %ro lang/erlang-runtime24
net/rabbitmq
```

```
$ pkg rquery %ro lang/erlang
net-im/ejabberd
lang/elixir
lang/erlang-wx
devel/erlang-recon
lang/erlang-java
devel/erlang-cl
devel/erlang-rebar3_hex
lang/lfe
devel/rebar
devel/rebar3
benchmarks/tsung
graphics/wings
www/yaws
```

I'm in the process of building them all, but it will be a little while until it's completed. So far so good!

File: lang/erlang/Makefile [hunk 1]
```diff
@@ -1,8 +1,7 @@
 PORTNAME=	erlang
 DISTVERSIONPREFIX=	OTP-
-DISTVERSION=	24.3.4.6
-PORTREVISION=	1
-PORTEPOCH=	4
```
Comment (dch) [inline lang/erlang/Makefile:5]: if we have an EPOCH, [it needs to be preserved](https://docs.freebsd.org/en/books/porters-handbook/book/#makefile-naming-revepoch)

File: lang/erlang/Makefile [hunk 2]
```diff
@@ -73,8 +72,7 @@ WX_DESC=	Enable WX application
 # probes fixed does not match the number of defined probes (54 != 132,
 # respectively)" you probably misconfigured DTrace in some way.
 
-CHUNKS_MAKE_ARGS+=		DOC_TARGETS=chunks \
-				docs
+CHUNKS_MAKE_ARGS+=		docs DOC_TARGETS=chunks
```
Comment (dch) [inline lang/erlang/Makefile:75]: no need to change this, unless `portfmt -t -i Makefile` disagrees *and* doesn't break thiings.

**Review by dch — CHANGES_REQUESTED** [review]
LGTM, fix the portepoch, and pending poudriere test, let's ship it.

File: lang/erlang/Makefile (hunk 1 above)
Comment (sax) [reply lang/erlang/Makefile:5]: Oh! I think I saw those docs earlier, but misread them as applying to the DISTVERSION. I just pushed a commit that brings it back.

File: lang/erlang/Makefile (hunk 2 above)
Comment (sax) [reply lang/erlang/Makefile]: Ok, sounds good. I saw this change in `lang/erlang-runtime25` and applied it, but it makes sense to me to keep this changeset as minimal as possible.

**Comment by dch** [comment]
thanks @sax, committed along with:

commit c9033971719dc098b34276f11b8e5a5e22b966fd
Author: Dave Cottlehuber <dch@FreeBSD.org>
Date:   Wed Jan 25 19:42:29 2023 +0000

    net/rabbitmq: update to 3.11.7 & switch to OTP25

    - while here, pet with portfmt and portclippy

commit 8893a5541856e22a2518b4ded2a8797d0c8974e0
Author: Dave Cottlehuber <dch@FreeBSD.org>
Date:   Wed Jan 25 19:42:29 2023 +0000

    lang/elixir-devel: update to 1.14.3

commit 1fd735a7234415ee15c7db3ee8b1e170945d35a4
Author: Dave Cottlehuber <dch@FreeBSD.org>
Date:   Wed Jan 25 19:42:29 2023 +0000

    databases/couchdb3: bump after lang/erlang-runtime25

commit 52ef12461ce4facd23d79794a2015500170adb3d
Author: Eric Saxby <sax@livinginthepast.org>
Date:   Wed Jan 25 19:42:29 2023 +0000

    lang/erlang-runtime25: update to version 25.2.1

commit 0e6053dc8839f9a90153cb42fe8a9755c27cf1e3
Author: Eric Saxby <sax@livinginthepast.org>
Date:   Wed Jan 25 19:42:28 2023 +0000

    lang/erlang-runtime24: update to 24.3.4.8

commit 428d54e3d199c8f6eed319becc6f58fa4c1798c8
Author: Eric Saxby <sax@livinginthepast.org>
Date:   Wed Jan 25 19:42:28 2023 +0000

    lang/erlang: update to version 25.2.1

**Pull request closed without merging**